src/sagemaker/jumpstart/curated_hub/curated_hub.py

```diff
@@ -113,3 +155,193 @@ def delete_model(self, model_name: str, model_version: str = "*") -> None:
     def delete(self) -> None:
         """Deletes this Curated Hub"""
         return self._sagemaker_session.delete_hub(self.hub_name)
+
+    def _is_invalid_model_list_input(self, model_list: List[Dict[str, str]]) -> bool:
+        """Determines if input args to ``sync`` is correct."""
```

Comment:
Can you expand this docstring? It is a little difficult to follow what the intent of this method is vs what the implementation is doing